lib/rubocop/cop/sorbet/signatures/checked_true_in_signature.rb

```diff
@@ -18,13 +19,9 @@ module Sorbet
       #
       #   # good
       #   sig { void }
-      class CheckedTrueInSignature < RuboCop::Cop::Cop
+      class CheckedTrueInSignature < SignatureCop
```

Comment:
Actually does this cop even apply anymore? It seems directly related to `waffle-cone` which we are not using anymore. I guess we can 🔥 it.

Reply:
`checked` is still in Sorbet spec: https://sorbet.org/docs/runtime#checked-whether-to-check-in-the-first-place

It uses symbols instead of `true/false`, it needs to be updated but is still relevant.

Reply:
Right, I forgot about that. Then I guess the check should be updated as well as the error message. But it can stay like this for now. We can change it in another PR.